pkg/reconcile/pipeline/handler/collect/impl.go

```diff
@@ -131,6 +132,23 @@ func ProvisionedService(ctx pipeline.Context) {
 	}
 }
 
+func DirectSecretReference(ctx pipeline.Context) {
+	services, _ := ctx.Services()
```

Comment:
I see `ctx.Services()` implementation might return errors, any special reason this is being ignored?

Reply:
Fixed. 58aea3667d70a507d6f3f32d6ea0dda8d26cb465